_slides/06-1604746.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Its a rainy day , I want to go out for Biking!
```

Comment:
Great caption, but you need to stay back to attend the training 😄

Reply:
:)